src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -192,24 +189,13 @@ const ActionLog = () => {
         setCurrentPage(value);
     };
 
-    const handleActionSort = () => {
-        currentSort.clicked ? setCurrentSort({ "sort": "action_desc", "clicked": !currentSort.clicked }) : setCurrentSort({ "sort": "action_asc", "clicked": !currentSort.clicked })
+    const handleSort = (sortType) => (event) => {
+        event.preventDefault();
+        currentSort.clicked ? setCurrentSort({ "sort": `${sortType}_desc`, "clicked": !currentSort.clicked }) : setCurrentSort({ "sort": `${sortType}_asc`, "clicked": !currentSort.clicked })
     }
-
-    const handleObjectSort = () => {
-        currentSort.clicked ? setCurrentSort({ "sort": "object_desc", "clicked": !currentSort.clicked }) : setCurrentSort({ "sort": "action_asc", "clicked": !currentSort.clicked })
-    }
-
-    const handleUserSort = () => {
-        currentSort.clicked ? setCurrentSort({ "sort": "user_desc", "clicked": !currentSort.clicked }) : setCurrentSort({ "sort": "user_asc", "clicked": !currentSort.clicked })
-    }
-
-    const handleAdminSort = () => {
-        currentSort.clicked ? setCurrentSort({ "sort": "admin_desc", "clicked": !currentSort.clicked }) : setCurrentSort({ "sort": "admin_asc", "clicked": !currentSort.clicked })
-    }
-
-    const handleDateSort = () => {
-        currentSort.clicked ? setCurrentSort({ "sort": "date_desc", "clicked": !currentSort.clicked }) : setCurrentSort({ "sort": "date_asc", "clicked": !currentSort.clicked })
+    const renderName = (actionId) => {
+        const selectedUser = admins.data.admins.find(user => (user.user.id == actionId));
+        return (<div>{`${selectedUser.user.firstName} ${selectedUser.user.lastName}`}</div>)
```

Comment:
if you want to print the full name, use the `fullName()` function in utils.js `return( <div>{fullName(selectedUser.use)}</div>`